src/bootstrap/builder.rs

```diff
@@ -1041,15 +1041,11 @@ impl<'a> Builder<'a> {
             }
         }
 
-        // FIXME: Don't use LLD with MSVC if we're compiling libtest, since it fails to link it.
-        // See https://github.com/rust-lang/rust/issues/68647.
-        let can_use_lld = mode != Mode::Std;
-
-        if let Some(host_linker) = self.linker(compiler.host, can_use_lld) {
+        if let Some(host_linker) = self.linker(compiler.host, true) {
```

Comment:
I think we always pass true now, could we drop that arg as well?

Reply:
Test suite is still run without explicit linker because there are tests that check linker search etc.
Perhaps we could switch to LLD there as well, but it may require adjusting some tests.
I'll try to do it in a separate PR.